README.md

```diff
@@ -8,81 +8,82 @@
 [![Coverage](https://codecov.io/gh/Toloka/toloka-kit/branch/main/graph/badge.svg)](https://codecov.io/gh/Toloka/toloka-kit)
 [![GitHub Tests](https://github.com/Toloka/toloka-kit/workflows/Tests/badge.svg?branch=main)](//github.com/Toloka/toloka-kit/actions?query=workflow:Tests)
 
+[Toloka website](https://toloka.ai/?utm_source=github&utm_medium=site&utm_campaign=tolokakit) | [Documentation](https://toloka.ai/en/docs/toloka-kit/?utm_source=github&utm_medium=site&utm_campaign=tolokakit) | [Issue tracker](https://github.com/Toloka/toloka-kit/issues)
 
-[**<span style="color:red">Documentation</span>**](https://toloka.ai/en/docs/toloka-kit/?utm_source=github&utm_medium=site&utm_campaign=tolokakit)
 
-[Website](https://toloka.ai/?utm_source=github&utm_medium=site&utm_campaign=tolokakit) |
-[API Documentation](https://toloka.ai/en/docs/api/?utm_source=github&utm_medium=site&utm_campaign=tolokakit) |
-[Platform](http://platform.toloka.ai/?utm_source=github&utm_medium=site&utm_campaign=tolokakit)
 
+Toloka-Kit is a Python library for working with [Toloka API](https://toloka.ai/docs/api/concepts/about.html/?utm_source=github&utm_medium=site&utm_campaign=tolokakit). 
 
-Designed by engineers for engineers, Toloka lets you integrate an on-demand workforce directly into your processes. Our cloud-based crowdsourcing platform is a fast and efficient way to collect and label large data sources for machine learning and other business purposes.
+The API allows you to build scalable and fully automated human-in-the-loop ML pipelines, and integrate them into your processes. The toolkit makes integration easier. You can use it with Jupyter notebooks.
 
-Main advantages of Toloka:
-  - **Top-quality data** -  Collect and annotate training data that meets and exceeds industry quality standards thanks to multiple quality control methods and mechanisms available in Toloka.
-  - **Scalable projects** - Have any amounts of image, text, speech, audio, or video data collected and labeled for you by millions of skilled Toloka users across the globe.
-  - **Cost-efficiency** - Save time and money with this purpose-built platform for handling large-scale data collection and annotation projects, on-demand 24/7, at your own price and within your timeframe.
-  - **Free, powerful API** - Build scalable and fully automated human-in-the-loop machine learning pipelines with a powerful open API.
+* Support for all common Toloka use cases: creating projects, adding pools, uploading tasks, and so on.
+* Toloka entities are represented as Python classes. You can use them instead of accessing the API directly via HTTPS.
+* There’s no need to validate JSON files and work with them directly.
+* Support of both synchronous and asynchronous usages.
+* Streaming support: send and receive data to Toloka in real time. It’s useful when you need to pass data between two related projects: one for data labeling, and another for its validation.  
```

Comment:
Maybe we should add a little extra here, to tell users that post validation is not the only use case
```suggestion
* Streaming support: build complex pipelines that send and receive data to Toloka in real time. E.g. you can pass data between two related projects: one for data labeling, and another for its validation.
```
Also we could probably add a link to this example: https://github.com/Toloka/toloka-kit/tree/main/examples/6.streaming_pipelines

Reply:
Changed to:
>Streaming support: build complex pipelines which send and receive data in real time. For example, you can [pass data between two related projects](https://github.com/Toloka/toloka-kit/blob/main/examples/6.streaming_pipelines/streaming_pipelines.ipynb): one for data labeling, and another for its validation.

Added a link to a certain example instead of a repo with several of them, so that our reader sees an example right away, instead of needing to somehow choose which one to open.